ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/rpm.py

```diff
@@ -13,7 +13,12 @@
     Commands:
 
         "rpm -qa --qf <query format>"
-
+        "if command -v dpkg dpkg >/dev/null 2>&1; then "
```

Comment:
Its a bad idea to put verbatim command in the docstring, because you have to maintain them in 2 places.